scrapy/crawler.py

```diff
@@ -123,17 +161,43 @@ def _setup_crawler_logging(self, crawler):
             crawler.signals.connect(log_observer.stop, signals.engine_stopped)
 
     def stop(self):
+        """
+        Stops simultaneously all the crawling jobs taking place.
+
+        Returns a deferred that is fired when they all have ended.
+        """
         return defer.DeferredList([c.stop() for c in list(self.crawlers)])
 
     @defer.inlineCallbacks
     def join(self):
-        """Wait for all managed crawlers to complete"""
+        """
+        join()
+
+        Wait for all managed :attr:`crawlers` to complete.
```

Comment:
I think we should mention a Deferred is returned, because this method itself doesn't wait for all crawlers to complete.